apps/codeforafrica/src/components/ErrorHero/ErrorHero.js

```diff
@@ -0,0 +1,51 @@
+import { RichTypography, Section } from "@commons-ui/core";
+import React from "react";
+
+import TwoToneBackground from "@/codeforafrica/components/TwoToneBackground";
+
+const ErrorHero = React.forwardRef(function ErrorHero(props, ref) {
+  const { title, subtitle, sx, ...other } = props;
+
+  return (
+    <TwoToneBackground
+      sx={{ py: { xs: 7.5, md: 10, lg: 12.5 }, ...other?.sx }}
+      ref={ref}
+    >
+      <Section
+        sx={{
+          maxWidth: {
+            sm: "648px",
+            md: "912px",
+          },
+          px: { xs: 2.5, sm: 0 },
+          zIndex: 1,
+          ...sx,
+        }}
+      >
+        <RichTypography
+          sx={{
+            paddingBottom: 2.5,
+          }}
+          variant="h1"
+        >
+          {title}
+        </RichTypography>
+        <RichTypography
+          variant="body1"
+          sx={{
+            fontSize: {
+              xs: "1.5rem",
+              sm: "1.75rem",
+              md: "2rem",
+              lg: "2.25rem",
+            },
+          }}
```

Comment:
```suggestion
          variant="h3Light"
          }}
```

Reply:
@kilemensi is the variant `body1` or `h3Light`. I can see both.

Reply:
Where do you see both of them @kelvinkipruto ?

Reply:
I think it should probably be `body1` as `h3Light` looks smaller than in the design.

Reply:
The question is where do you get `body1` from @kelvinkipruto. I'm getting `h3Light` 👇🏽 on Figma.
![S](https://user-images.githubusercontent.com/1779590/185891917-834ba103-74fc-42da-8e8a-3393b95a92a3.png)